front/app/components/UI/Error/index.tsx

```diff
@@ -184,8 +184,8 @@ export const findErrorMessage = (
   if (messages[error]) {
     return messages[error] as Message;
   }
-  // Return a generic error message
-  return messages.invalid;
+  // Return a empty error message
+  return '';
```

Comment:
Before we were returning a default message but it was a very unclear one and it didn't make sense in many cases. This is now replaced with an empty to not confuse users but maybe we can improve it in the future.